test/libsolidity/syntaxTests/comments/singleline_unicode_direction_override_6.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Some extra test suggestions:
- Unmatched LRO/RLO in a file with an unterminated multi-line comment.
- Unmatched LRO/RLO in a single-line comment at the end of the file.

Including tests for stuff that should not affect your code if it works correctly:
- LRO/RLO/PDF outside of a comment (would that be a syntax error?):
    - I a random place in code.
    - In the name of an imported file.
    - In a string.
- A test including PDI, LRI/RLI/LRE/RLE just to show they don't affect the output (or more tests if you're going to change the code so that they do).
- A test including line and paragraph end characters.

Reply:
@mijovic I see that you added some of these. Could you add he rest? Especially the ones for unterminated comments or comments at the end of file.